examples/webgl_materials_lightmap.html

```diff
@@ -96,32 +66,25 @@
 
 				// SKYDOME
 
-				const vertexShader = document.getElementById( 'vertexShader' ).textContent;
-				const fragmentShader = document.getElementById( 'fragmentShader' ).textContent;
-				const uniforms = {
-					topColor: { value: new THREE.Color( 0x0077ff ) },
-					bottomColor: { value: new THREE.Color( 0xffffff ) },
-					offset: { value: 400 },
-					exponent: { value: 0.6 }
-				};
-				uniforms.topColor.value.copy( light.color );
-
-				const skyGeo = new THREE.SphereGeometry( 4000, 32, 15 );
-				const skyMat = new THREE.ShaderMaterial( {
-					uniforms: uniforms,
-					vertexShader: vertexShader,
-					fragmentShader: fragmentShader,
-					side: THREE.BackSide
-				} );
-
-				const sky = new THREE.Mesh( skyGeo, skyMat );
+				const topColor = new THREE.Color().copy( light.color ).convertSRGBToLinear();
+				const bottomColor = new THREE.Color( 0xffffff ).convertSRGBToLinear();
+				const offset = 400;
+				const exponent = 0.6;
+
+				const h = positionLocal.add( offset ).normalize().y;
+
+				const skyMat = new MeshBasicNodeMaterial();
+				skyMat.colorNode = vec4( mix( color( bottomColor ), color( topColor ), max( pow( max( h, 0.0 ), exponent ), 0.0 ) ), 1.0 );
```

Comment:
Second `max` is not needed, because `pow( max( h, 0.0 ), exponent )` would be always positive (also, maybe we can rewrite it as `h.max( 0.0 ).pow( exponent )`)?